### PR TITLE
Coordinated datalist and highlight colors for search, effects, appear…

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
Refers to issue #3628 and #3739 

All the dialogs should have the same background colors and outlines: 

Final datalist dialog: 

Final keyboard shortcut dialog: 

Final export dialog: 

Final introduction screen dialog:

### CODE REVIEW

**Comment by placegraphichere** [comment]
So the `var` we're using is for "text highlighting". But in this instance we are trying to set the color of the text itself. I think we need a new color for the text: color. 

The contrast for the background color on datalists is also problematic trying to find a color that suits the darker color but is useful elsewhere in the app for that color theme.

I'll add some more info on this shortly.

**Comment by DivyaPrabhakar** [comment]
@placegraphichere When you hover over datalist options, isn't that highlighting the text? I am confused about what you mean by we are "setting the color of the text" -- can you explain?

**Comment by iwehrman** [comment]
I think this is on @placegraphichere's plate currently, so assigning to him.

File: src/style/search/search-bar.less [hunk 1]
```diff
@@ -29,10 +29,11 @@
     background: no-repeat left @bg-panel;
     background-size: @text-xxlarge;
     background-position: 1.7rem 1.5rem;
-    top: 25%;
-    left: 50%;
+    top: round(25%);
```
Comment (iwehrman) [inline src/style/search/search-bar.less]: This looks suspicious to me. I would guess that this is equivalent to `25%`, but I'm not sure. Registering this concern for posterity.

**Comment by iwehrman** [comment]
The main blocker is that the search bar is not correctly positioned in this branch. It seems to be positioned in the middle of the screen instead of in the middle of the canvas:
image

**Comment by iwehrman** [comment]
Colors look great. Back to @DivyaPrabhakar to fix the positioning issue.

File: src/style/search/search-bar.less (hunk 1 above)
Comment (designedbyscience) [inline src/style/search/search-bar.less]: Agreed...this is a LESS function which doesn't seem to make it into the final CSS. It is translated to 25%...

**Comment by iwehrman** [comment]
I've amended the commit to fix the positioning problem, which I think was due to repeated `transform` rules. I think this is ready to merge, but I'm unassigning myself so that someone else can double-check it.

**Comment by placegraphichere** [comment]
I'm still seeing the gap between the bar and edge of the hovered item and the search input. I thought we got this sorted, but it's still not. I believe this is due to the overall dialog positioning being on a half px.

screen shot 2016-02-10 at 5 37 19 pm

screen shot 2016-02-10 at 5 39 00 pm

It'd be great to just round this position to a whole px. Otherwise, this is all great stuff.

**Comment by placegraphichere** [comment]
So on a retina, if you just resize the app frame to be slightly smaller on an odd pixel (1907x 1178 was what I resized it to), usually you'll see this occur. If it's full screen, it will look correct.

**Comment by iwehrman** [comment]
AFAICT this is back in @DivyaPrabhakar's court.

**Comment by iwehrman** [comment]
FWIW, I can't repro this issue. I tried to get the window as close as possible to the resolution that @placegraphichere suggested, but it still looks good to me.

**Comment by ktaki** [comment]
Tried with various resolutions and app windows sizes on MacBook Pro retina. I do not see the gap @placegraphichere mentioned either.

**Comment by DivyaPrabhakar** [comment]
@placegraphichere Can you can sign off on the changes to the keyboard shortcut dialog? After that, I can offer this PR up to the team so someone can review and merge it.

File: src/style/shared/colors-medium.less [hunk 2]
```diff
@@ -59,6 +60,7 @@
 
 @focus-highlight: #3691ff;
 @text-highlight: #1B8BE9;
+@text-focus: #1473e6;
```
Comment (volfied) [inline src/style/shared/colors-medium.less]: For code cleanliness sake, can you make sure hex letters are uppercase like the examples above?

File: src/style/shared/colors-original.less [hunk 3]
```diff
@@ -26,6 +26,7 @@
 @bg-border: #CCCCCC;
 @bg-panel: #f0f0f0;
 @bg-alt: #DBDBDB;
+@bg-dialog: #FFF;
```
Comment (volfied) [inline src/style/shared/colors-original.less]: Is this short hand for FFFFFF or F0F0F0 or 0F0F0F? You get my point :)

**Comment by DivyaPrabhakar** [comment]
@volfied style changes made

**Comment by iwehrman** [comment]
LGTM!

**Pull request merged**